src/bindings/python/tests/test_graph/test_ops_binary.py

```diff
@@ -216,13 +220,16 @@ def test_binary_bitwise_op(graph_api_helper, dtype, shape_a, shape_b, broadcast,
 
 @pytest.mark.parametrize(
     "graph_api_helper",
-    [ov.bitwise_and, ov.bitwise_or, ov.bitwise_xor],
+    [ov.bitwise_and, ov.bitwise_or, ov.bitwise_xor, ov_opset15.bitwise_left_shift, ov_opset15.bitwise_right_shift],
 )
 @pytest.mark.parametrize(
     "dtype",
     [bool, np.int32],
 )
 def test_binary_bitwise_op_with_constant(graph_api_helper, dtype):
+    if "shift" in graph_api_helper.__name__ and dtype == bool:
+        pytest.skip(f"The element type of the input tensor for {graph_api_helper.__name__} must be integer number.")
```

Comment:
Usually we use `skip` in case of some limitation or bug in the product.
Please create separate test for binary shift operations and parameterize parameter and constant case by using `is_const` test parameter, for example. Also, I am not sure that negative constant value (below) is valid for shift usage.